test/k8s-integration/cluster.go

```diff
@@ -345,7 +345,7 @@ func clusterUpGKE(gceZone, gceRegion string, numNodes int, imageType string, use
 		if err != nil {
 			return fmt.Errorf("failed to Create kubernetes e2e cluster on gke: %v", err)
 		}
-		err = waitForOp(service.Projects.Locations.Operations, parent, op)
+		err = waitForOp(service.Projects.Locations.Operations, parent, op, 10)
```

Comment:
Add a constant, eg to driver-config.go where we have the pod timeouts (or maybe just to this file since it's cluster-specific)

Reply:
Done